.github/workflows/ci.yml

```diff
@@ -49,10 +49,18 @@ jobs:
           git clone https://github.com/$GITHUB_REPOSITORY $GITHUB_WORKSPACE --no-checkout --no-tags
           git checkout $GITHUB_SHA
           git submodule update --init --depth 1000 --jobs 8
-      - name: Building with bazel
+      - name: Building and testing with bazel
         run: |
           source ~/vulkan/1.1.126.0/setup-env.sh
-          bazel build //...
-      - name: Testing with bazel
-        # TODO(gcmn) Enable all tests once they're passing
-        run: bazel test --test_output=errors -- //... -//integrations/... -//bindings/... -//iree/hal/vulkan:dynamic_symbols_test -//iree/samples/rt:bytecode_module_api_test
```

Comment:
Don't we still want to exclude these targets?

`-//iree/hal/vulkan:dynamic_symbols_test -//iree/samples/rt:bytecode_module_api_test`

Those tests use Vulkan and need either a physical GPU on the test machine or for SwiftShader to be configured (installing the Vulkan SDK should not be necessary or sufficient, that just enabled static linking in `iree/samples/rt/vulkan:vk_graphics_integration`). SwiftShader is configured upstream, hence no "notap" tag being set. If they build and pass without SwiftShader then I'd be curious to know how :P

Reply:
Oh whoops, thanks. I missed those because they built on my local box, but that's because I have a GPU I guess :-D